raiden-ts/src/raiden.ts

```diff
@@ -955,6 +956,29 @@ export class Raiden {
       )
       .toPromise();
   }
+
+  /**
+   * Mints the amount of tokens of the provided token address.
+   * Throws an error if executed on main net.
+   *
+   * @param tokenAddress - Address of the token to be minted
+   * @param amount - Amount to be minted
+   * @returns transaction
+   */
+  public async mint(tokenAddress: string, amount: BigNumberish): Promise<ContractTransaction> {
+    // Check whether address is valid
+    if (!Address.is(tokenAddress)) throw new Error('Invalid address');
+
+    // Check whether we are on a test network
+    const { name } = await this.deps.provider.getNetwork();
+    if (!['goerli', 'rinkeby', 'ropsten'].includes(name)) {
```

Comment:
If ever 4th testnet comes to life code won't work. Why don't you just look for mainnet and then it will always work?

Reply:
The SDK will fail much earlier if a new test net comes along. But I guess it's worth having to change one part less of the code in that case. Will fix